client/app/domain/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -58,7 +58,8 @@ <h3 class="oui-tile__title" data-i18n-static="domain_dashboard_general_informati
                                                         data-ng-if="!ctrlTabGeneralInformations.dnsStatus.isOk">
                                                 </button>
                                                 <small class="ml-3"
-                                                       data-ng-if="ctrlTabGeneralInformations.dnsStatus.refreshAlert">
+                                                       data-ng-if="ctrlTabGeneralInformations.dnsStatus.refreshAlert"
+                                                       data-uib-tooltip="{{::i18n.domain_dashboard_tooltip_dns_updating}}">
```

Comment:
`data-uib-tooltip-append-to-body="true"` isn't required here?

Reply:
Nope